tests/base/__init__.py

```diff
@@ -52,14 +53,22 @@ def setUp(self):
         """Before each test case, setup a dummy tracer to use"""
         self.tracer = DummyTracer()
 
+        # swap out global tracer
+        self.global_tracer = ddtrace.tracer
+        ddtrace.tracer = self.tracer
```

Comment:
This was necessary since `correlation.get_correlation_ids()` used by the `logging` patch method inserts trace information using it.

Reply:
oo... this might cause some issues in other tests... I am willing to bet.

Reply:
Only `tests/test_tracer.py` and `tests/utils/span.py` currently use the `BaseTracerTestCase` so the effect is not as big.

Reply:
until you review one of the PRs I have open 😆

Reply:
but willing to give it a try